params/protocol_params.go

```diff
@@ -170,6 +170,7 @@ const (
 	Bls12381MapG2Gas          uint64 = 75000 // Gas price for BLS12-381 mapping field element to G2 operation
 
 	P256VerifyGas uint64 = 3450 // secp256r1 elliptic curve signature verifier gas price
+	GpgEd25519VerifyGas uint64 = 2000 // GPG Ed25519 signature verification gas price
```

Comment:
We should do some benchmarking to make sure this price is still appropriate, given the extra computation.

Reply:
Plan for this:
- run other precompile benchmarks to get MGas/s on local machine
- run our precompile to get a sense of runs/s with varying inputs 
- create pricing formula that leads to MGas/s equal or greater than other precompiles for all inputs

Reply:
I'd also like to get a sense of whether there are codepaths in the Go libs that are very time intensive that could be abused as a DOS vector. We can poke around a bit, but also will leave a note for auditors to explore this.